lib/terraform.go

```diff
@@ -52,17 +53,53 @@ func NewContext(dir, varFile string, resolver providers.Resolver) (*terraform.Co
 		variables = InputValuesFromType(values, terraform.ValueFromNamedFile)
 	}
 
+	// Bind available provider plugins to the constraints in config
```

Comment:
I stole this from the old terraform code

Reply:
Probably should remove this line. I'm feeling I don't understand enough of what this is doing

Reply:
![](https://media.tenor.com/images/00760d23371272db216da7919ce7c884/tenor.gif)

Reply:
However it is working but probably needs to be looked over